### PR TITLE
Adding names

### PR DESCRIPTION
**Pull request by kevinprince**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # training
 how we use github demo
- 
+
+
+Hello Fahran
```
Comment (kevinprince) [inline README.md]: This should probably be @emson

**Comment by fahran** [comment]
:+1: :shipit:

**Comment by fahran** [comment]
:+1: :shipit:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # training
 how we use github demo
- 
+
```
Comment (kevinprince) [inline README.md]: Try this instead

```ruby
  def name
   'joycey'
  end 

  print name
```

**Pull request merged**